pkg/installer/custominstallconfig.go

```diff
@@ -203,3 +211,28 @@ func appendFilesToCvoOverrides(a asset.WritableAsset, g graph.Graph) (err error)
 
 	return nil
 }
+
+// replacePointerIgnition performs the same functionality as the upstream
+// installer's pointerIgnitionConfig() but with ARO specific DNS config
+func replacePointerIgnition(a asset.WritableAsset, g graph.Graph, aroDNSConfig *bootkube.ARODNSConfig) (err error) {
```

Comment:
If you rebase this PR now you'll be able to use a `dnsmasq.DNSConfig` instead of `bootkube.ARODNSConfig` (the latter is only part of the fork and is going away when we switch to upstream).